integration/src/main/java/scripts/testscripts/SnapshotDatasetOperations.java

```diff
@@ -188,7 +189,7 @@ private void resultHasDatasetWithRoles(List<Object> datasets) {
         assertThat(dataset, hasEntry(is("id"), is(datasetId.toString())));
         @SuppressWarnings("unchecked")
         List<Object> roles = (List<Object>) dataset.get("roles");
-        assertThat(roles, containsInAnyOrder("steward", "admin"));
+        assertThat(roles, hasSize(not(0)));
```

Comment:
Does this work?
```suggestion
        assertThat(roles, not(empty()));
```

Or we could assert the one element we know will be there
```suggestion
        assertThat(roles, hasItem("steward"));
```

Reply:
Unlike the preview, the content of the roles should be asserted since our code depends on it not changing over time.

Reply:
So, in that case, we need to make sure we are either not in, or are in, the DataRepoAdmins group. I assume we should talk to Jade Core to get our admin account added to that group in all envs?

Reply:
… nevermind, that's not actually blocking. How did I miss that hasItem isn't a complete equivalence.